server/statistics/hot_regions_stat.go

```diff
@@ -15,8 +15,10 @@ package statistics
 
 // HotPeersStat records all hot regions statistics
 type HotPeersStat struct {
-	TotalBytesRate float64       `json:"total_flow_bytes"`
-	TotalKeysRate  float64       `json:"total_flow_keys"`
-	Count          int           `json:"regions_count"`
+	StoreBytesRate float64       `json:"store-bytes-rate"`
```

Comment:
Why does hot peer stats need to have StoreBytesRate

Reply:
emmm, its name is confusing.